docs/howto/composite-primary-key.txt

```diff
@@ -0,0 +1,161 @@
+=====================================
```

Comment:
To me, this document seems like it should be in 'topics' rather than 'how-to'.

Reply:
Agreed, @csirmazbendeguz do you mind making this update?

Reply:
I moved it 👍